app/controllers/chargeB/ChargeDetailsController.scala

```diff
@@ -51,20 +49,14 @@ class ChargeDetailsController @Inject()(override val messagesApi: MessagesApi,
                                         formProvider: ChargeDetailsFormProvider,
                                         val controllerComponents: MessagesControllerComponents,
                                         deleteChargeHelper: DeleteChargeHelper,
-                                        renderer: Renderer)(implicit ec: ExecutionContext)
+                                        view : ChargeDetailsView)(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport
     with NunjucksSupport {
```

Comment:
sorry forgot to comment on this one
Please remove this with NunjucksSupport and import

Reply:
done